test/runtime/samples/inline-style-directive-and-style-attr/main.svelte

```diff
@@ -0,0 +1,5 @@
+<script>
+	export let color = 'red';
+</script>
+
+<p style="height: 40px; color: blue;" style:color={color} />
```

Comment:
@tanhauhau You had asked about precedence — the style directive will always take precedence (which was discussed a bit in the RFC)